### PR TITLE
[dev-1.30] docs: Update KubeletConfigDropinDir doc information

### PR DESCRIPTION
**Pull request by haircommander**
taking over https://github.com/kubernetes/website/pull/45479/ as @sohankunkerkar is on vacation

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (mrunalp) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:99]: Not sure what this line is conveying.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (mrunalp) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:106]: Files earlier within that sort.. ?

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (mrunalp) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:260]: exclude -> excluding

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (mrunalp) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:272]: Maybe add motivation for this:  `Since the configuration could now be spread over multiple files with this feature, if someone wants to inspect the final actuated configuration, they can follow these steps: `

**Comment by haircommander** [comment]
thanks @mrunalp updated

**Review by mrunalp — APPROVED** [review]
/approve
sig-node

**Comment by natalisucks** [comment]
/lgtm
from docs, thanks @haircommander for opening this new PR to get in before the docs freeze!
@drewhagen @katcosgrove this PR replaces #45479 due to the original author being on vacation, and I can also see @mrunalp has given their tech approval from sig node.
if the replacement is fine for you, please go ahead and approve 🙏

**Comment by drewhagen** [comment]
Thank you @natalisucks for the courteous heads up!

/lgtm

@haircommander I really appreciate you creating a new PR to get in before doc freeze while another is on vacation. Before I approve, I'm curious, how does this PR relate to https://github.com/kubernetes/website/pull/44985? Does that also need to get in?

Thanks in advance for the clarity!

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:112]: Will readers know what these are? (my guess is: they won't)

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:157]: Will readers know what we mean here? In YAML, they're called “mappings”.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:110]: The details around configuration merging are specific and aren't right for a task page. Please move them into https://kubernetes.io/docs/reference/node/ so that people wanting to configure the kubelet can still understand the overall task.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:280]: We [separate commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:287]: We [separate commands from output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)

Also, make sure that `jq` and `curl` are listed as task prerequisites.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:100]: Optionally:
```suggestion
The suffix of a valid kubelet drop-in configuration file **must** be `.conf`. For instance: `99-kubelet-address.conf`
```

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:104]: (nit)
```suggestion
For instance, `00-kubelet.conf` is processed first, and then overridden with a file named `01-kubelet.conf`.
```

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:107]: (nit)
```suggestion
Validation is only performed on the final resulting configuration document.
```

**Comment by sftim** [comment]
I see this has LGTM.

I'm minded not to approve it based on my own [review](https://github.com/kubernetes/website/pull/45665#pullrequestreview-1960170326) (I think we're making an existing task hard to follow, and shouldn't be doing that), but I'm happy to for a different approver for English add their approval.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (haircommander) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md:157]: I don't see any reference to mappings, only Lists

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md:157]: If we're talking about lists, let's just call them lists.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (haircommander) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md:107]: I think structure is clearer actually

**Comment by haircommander** [comment]
thanks @sftim updated

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (divya-mohan0209) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md:164]: @haircommander / @sftim : Can we take this up in a followup PR?

**Review by divya-mohan0209 — COMMENTED** [review]
Overall, I think this looks good, and I'd like to bias for action.

I'd recommend that we address [Tim's comment](https://github.com/kubernetes/website/pull/45665/files#r1539104466) over a separate followup PR with any other outstanding concerns.

/approve

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md
Comment (haircommander) [reply content/en/docs/tasks/administer-cluster/kubelet-config-file.md:164]: It was done above https://github.com/kubernetes/website/pull/45665#pullrequestreview-1962151298 :)

**Pull request merged**
